sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/AzureMonitorAspNetCoreEventSourceTests.cs

```diff
@@ -13,7 +13,7 @@ public class AzureMonitorAspNetCoreEventSourceTests
         /// This validates that parameters are logged and helps to confirm that EventIds are correct.
         /// </summary>
         [Fact]
-        public void EventSourceTest_AzureMonitorExporterEventSource()
```

Comment:
misc change. This file was copied from Exporter to AspNetCore but wasn't renamed. I caught this when copying into the LiveMetrics project.